javascript/javascript1/week3/item-array-removal/item-array-removal.js

```diff
@@ -0,0 +1,18 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+for (let i = 0; i < names.length; i++) {
+  if (names[i] !== -1 && names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
```

Comment:
This is the most logical way to make it. However, there is an array method which can make it much easier. It's called [`indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf).
Using it for this exercise would be something like this:
```
const indexOfNameToRemove = names.indexOf(nameToRemove);

if (indexOfNameToRemove >= 0) {
  names.splice(indexOfNameToRemove, 1);
} else {
  console.log("Informative error message");
}
```